docs/faqs/max_execution_time.rst

```diff
@@ -1,74 +1,85 @@
 
-IBM Quantum system limit
-*****************************
+.. code-block:: python
 
-The system limit on the Qiskit Runtime job execution time is described in the following table:
+   # Initiate the Options class with parameters
+   options = Options(max_execution_time=360)
 
-+------------------+--------------+------------------+--------------+-----------+
-|                  | Primitive programs              | Private programs         |
-+==================+==============+==================+==============+===========+
-|                  | Premium User | Open User        | Premium User | Open User |
-+------------------+--------------+------------------+--------------+-----------+
-| Simulated Device | 3h           | 1h               | 3h           | N/A       |
-+------------------+--------------+------------------+--------------+-----------+
-| Real Device      | 8h           | 4h               | 8h           | N/A       |
-+------------------+--------------+------------------+--------------+-----------+
+.. code-block:: python
 
-Note that a *premium user* here means a user who has access to backends in providers other than ibm-q/open/main.
+   # Create the options object with attributes and values
+   options = {"max_execution_time": 360}
 
-Program time limits
-***************************
+You can also find quantum time used by previously completed jobs by using:
 
-Different types of programs have their own time limits:
+.. code-block:: python
 
-* **Primitives:** The maximum execution time for the Sampler primitive is 10000 seconds (2.78 hours). The maximum execution time for the Estimator primitive is 18000 seconds (5 hours).
-* **Private Programs:** The maximum execution time (in seconds) for a program is set on the job options with the ``max_execution_time`` parameter. 
+   # Find quantum time used by the job
+   print(f"Quantum time used by job {job.job_id()} was {job.metrics()['usage']['quantum_seconds']} seconds")
 
-Job max execution time
+In addition, the system calculates an appropriate job timeout value based on the
+input circuits and options. This system-calculated timeout is currently capped
+at 3 hours to ensure fair usage of the devices. If a ``max_execution_time`` is
+also specified for the job, the lesser of the two values is used.
+
+For example, if you specify ``max_execution_time=5000``, but the system determines
+it should not take more than 5 minutes (300 seconds) to execute the job, then the job will be
+cancelled after 5 minutes.
+
+Session time limits
 ***************************
 
-Set the maximum execution time (in seconds) on the job options by using one of the following methods.  The value must be **300 or higher**:
+When a session is started, it is assigned a maximum session timeout value.
+After this timeout is reached, the session is terminated and any queued jobs that remain in the session are put into a ``failed`` state.
```

Comment:
Should we mention that any jobs that are running are also failed?  And should we mention the specific error you'd see?

Reply:
Excellent question. I can't remember if we actually cancel running jobs. Let me double check and get back to you.

Reply:
ok verified that running jobs don't get cancelled